ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -208,6 +208,7 @@ cHeaders :: GhcFlavor -> [String]
 cHeaders ghcFlavor =
   [ "includes/ghcconfig.h"
   , "includes/MachDeps.h"
+  , "includes/stg/MachRegs.h"
```

Comment:
Shouldn’t this change depend on the STG version? I don’t see how this could have changed for older versions.

Reply:
Not an oversight. The inclusion of a new set of constants to support the LLVM backend highlighted a bug in our inclusion scheme. The contents of `includes/CodeGen.Platform.hs` includes ` stg/MachRegs.h`. The former is in the `extra-sources` of ghc-lib targets but not the latter which we've been satisfying by provision from the compiler. Now we include `MachRegs.h` (as we should) along with `MachDeps.h`.

Reply:
Sounds good :+1: